src/console/plugin.ts

```diff
@@ -105,7 +109,7 @@ interface ICreateConsoleArgs extends JSONObject {
  * Activate the console extension.
  */
 function activateConsole(app: JupyterLab, services: IServiceManager, rendermime: IRenderMime, mainMenu: IMainMenu, inspector: IInspector, palette: ICommandPalette, pathTracker: IPathTracker, renderer: ConsoleContent.IRenderer): IConsoleTracker {
-  let tracker = new FocusTracker<ConsolePanel>();
+  let tracker = new FocusTracker<Widget>();
```

Comment:
Yeah, I rethought this approach and now I'm not using a `FocusTracker` at all ... because it's not really what we need here.